packages/fela/src/getDocumentRefIndex.js

```diff
@@ -0,0 +1,17 @@
+import arrayReduce from 'fast-loops/lib/arrayReduce'
+
+import type { DOMRendererDocumentRef } from '../../../flowtypes/DOMRenderer'
+
+const getDocumentRefIndex = (
```

Comment:
Can be replaced with `.findIndex()` but there is no such function in `fast-loops` and it's not supported by IE11